lib/errorlogger.cpp

```diff
@@ -637,7 +637,9 @@ std::string ErrorMessage::toString(bool verbose, const std::string &templateForm
         findAndReplace(result, replaceFrom, replaceWith);
         pos1 = result.find("{inconclusive:", pos1);
     }
-    findAndReplace(result, "{severity}", severityToString(severity));
+    std::string sseverity = coloredSeverityToString(severity);
```

Comment:
This would introduce coloring even if it has been disabled. If you provide a custom template you need to provide the colors yourself.

It seems like this is actually a missing feature being able to specify colored severities. So this probably needs to be a separate macro.